src/card.css

```
@@ -0,0 +1,91 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-accent-color);
+}
+
+.card__vendor-code {
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+  color: var(--text-accent-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  line-height: 18px;
+  font-size: 12px;
+  color: var(--text-secondary-color);
+}
+
+.card__price :last-child {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--text-accent-color);
+}
+
+.card__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+
+  background: var(--main-color);
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+}
+
+.card__link--hover:hover {
```

Don’t need create new class for hover (you can use card__link class)

```suggestion
.card__link:hover {
```